packages/scaffolding-cli/templates/src/ssr/server/api/authRoutes.ts

```diff
@@ -29,11 +31,18 @@ export default (router: Router) => {
         })(req, res, next)
     })
 
-    router.get("/logout", (req: any, res) => {
+    router.get("/logout", (req: any, res: any) => {
+        removePassportCookie(res, "connect.sid")
         req.logout()
         const {AUTH0_DOMAIN, AUTH0_CLIENT_ID, AUTH0_BASE_URL} = process.env
         res.redirect(
             `https://${AUTH0_DOMAIN}/logout?client_id=${AUTH0_CLIENT_ID}&returnTo=${AUTH0_BASE_URL}`,
         )
+        res.end()
+    })
+
+    router.get("/user", (req: any, res) => {
+        // const session = cookie.parse(req.headers?.cookie || "")
+        res.status(200).json({user: req.user || null})
```

Comment:
Catch any errors?

Reply:
We do not want to throw any 500s here. The UI components handles all the user=null (fallback) scenarios.